examples/notebooks/beam-ml/run_inference_tensorflow_hub.ipynb

```diff
@@ -0,0 +1,307 @@
+      "source": [
+        "import os\n",
+        "import importlib\n",
+        "\n",
+        "import numpy as np\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_hub as hub\n",
+        "import tensorflow_text\n",
+        "\n",
+        "from tensorflow import keras\n",
+        "\n",
+        "from typing import Any\n",
+        "from typing import Dict\n",
+        "from typing import Iterable\n",
+        "from typing import Optional\n",
+        "from typing import Sequence\n",
+        "\n",
+        "import apache_beam as beam\n",
+        "import apache_beam.runners.interactive.interactive_beam as ib\n",
+        "\n",
+        "from apache_beam.ml.inference.base import RunInference\n",
+        "from apache_beam.ml.inference.base import ModelHandler\n",
+        "from apache_beam.ml.inference.base import PredictionResult\n",
+        "from apache_beam.internal import pickler\n",
+        "from apache_beam.runners.runner import PipelineResult\n",
+        "from apache_beam.runners.interactive.interactive_runner import InteractiveRunner\n",
+        "\n",
+        "from tfx_bsl.public.beam.run_inference import CreateModelHandler\n",
+        "from tfx_bsl.public.proto import model_spec_pb2"
+      ],
+      "id": "f313a508-59ea-47ed-86eb-c9c8e67785f2"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+      },
+      "source": [
+        "## Create a Keras Model from TensorFlow Hub Image"
+      ],
+      "id": "74db0203-3d26-4bc4-8271-81fad9756297"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "r1fgn5I_-mzA"
+      },
+      "outputs": [],
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "id": "r1fgn5I_-mzA"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2ff8e394-f577-4dea-bef9-a4f4528c1378"
+      },
+      "outputs": [],
+      "source": [
+        "PROJECT_ID = '<Project Id>'\n",
+        "GCS_BUCKET = '<GCS Bucket>'\n",
+        "\n",
+        "MODEL_PATH = f'{GCS_BUCKET}/st5-base/1'"
+      ],
+      "id": "2ff8e394-f577-4dea-bef9-a4f4528c1378"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ccaede25-1c1a-4ec4-9296-25c9a2ac43d7"
+      },
+      "outputs": [],
+      "source": [
+        "inp = tf.keras.layers.Input(shape=[], dtype=tf.string, name='input')\n",
+        "hub_url = \"https://tfhub.dev/google/sentence-t5/st5-base/1\"\n",
+        "imported = hub.KerasLayer(hub_url)\n",
+        "outp = imported(inp)\n",
+        "model = tf.keras.Model(inp, outp)\n",
+        "\n",
+        "# Sentenced-T5 model returns a 768-dimensional vector for an English text input.\n",
+        "# Note the 'input' that we will pass in as example's feature key name.\n",
+        "model.summary()"
```

Comment:
Move the model.summary to a different cell for better visibility.

Reply:
Done.